content/engine/release-notes/25.0.md

```diff
@@ -0,0 +1,103 @@
+- Automatically enable IPv6 on a network when an IPv6 subnet is specified  [moby/moby#46455](https://github.com/moby/moby/pull/46455)
+- Configuration reloading is now more robust: if there's an error during the configuration reload process, no configuration changes are applied. [moby/moby#43980](https://github.com/moby/moby/pull/43980)
+- Ensure network names are unique. [moby/moby#46251](https://github.com/moby/moby/pull/46251)
+- Ensure that overlay2 layer metadata is correct. [moby/moby#46471](https://github.com/moby/moby/pull/46471)
+- Fix `Downloading` progress message on image pull. [moby/moby#46515](https://github.com/moby/moby/pull/46515)
+- Fix `NetworkConnect` and `ContainerCreate` with improved data validation, and return all validation errors at once. [moby/moby#46183](https://github.com/moby/moby/pull/46183)
+- Fix `com.docker.network.host_ipv4` option when IPv6 and ip6tables are enabled. [moby/moby#46446](https://github.com/moby/moby/pull/46446)
+- Fix daemon's `cleanupContainer` if containerd is stopped. [moby/moby#46213](https://github.com/moby/moby/pull/46213)
+- Fix returning incorrect HTTP status codes for libnetwork errors. [moby/moby#46146](https://github.com/moby/moby/pull/46146)
+- Fix various issues with images/json API filters and image list. [moby/moby#46034](https://github.com/moby/moby/pull/46034)
+- Set the MAC address of container's interface  when network mode is a short network ID  [moby/moby#46406](https://github.com/moby/moby/pull/46406)
+- Sort unconsumed build arguments before display in build output. [moby/moby#45917](https://github.com/moby/moby/pull/45917)
+- The `docker image save` tarball output is now OCI compliant. [moby/moby#44598](https://github.com/moby/moby/pull/44598)
+- The daemon no longer appends `ACCEPT` rules to the end of the `INPUT` iptables chain for encrypted overlay networks. Depending on the system's firewall configuration, a rule such as `iptables -A INPUT --dir in -p udp --dport 4789 -m policy --pol ipsec -j ACCEPT` may need to be programmed to permit incoming encrypted overlay network traffic. [moby/moby#45280](https://github.com/moby/moby/pull/45280)
+- Unpacking layers containing files with extended attributes onto a filesystem which doesn't support extended attributes will now fail. Previously, the extended attributes would be discarded if they couldn't be set, which could potentially result in containers malfunctioning. Most storage drivers already require the backing filesystem to support extended attributes so this change should have no impact on most users. The `vfs` driver is a notable exception; the previous behaviour can be restored by configuring it with `--storage-opt vfs.xattrs=i_want_broken_containers`. Enabling this storage option is highly discouraged and bug reports submitted against a daemon with this option set will be summarily closed without comment. [moby/moby#45464](https://github.com/moby/moby/pull/45464)
```

Comment:
bit long, can we keep only the first sentence?

Reply:
```suggestion
- Unpacking layers with extended attributes onto an incompatible filesystem will now fail instead of silently discarding extended attributes. [moby/moby#45464](https://github.com/moby/moby/pull/45464)
```